wsd/FileServer.cpp

```diff
@@ -1753,7 +1753,7 @@ FileServerRequestHandler::ResourceAccessDetails FileServerRequestHandler::prepro
 
     // default to the notebookbar if the value is "default" or whatever
     // nonsensical
-    if (enableAccessibility == "true" || (userInterfaceMode != "classic" && userInterfaceMode != "notebookbar"))
+    if (enableAccessibility == "true" && userInterfaceMode != "classic" && userInterfaceMode != "notebookbar")
```

Comment:
I don't understand this change. You changed the logic, but why?

Reply:
If enableAccessbility is true, no way to change the user interface to 'compact', it will always force to interface .